specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/examples/ManagedDatabaseSchemaGet.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "myRG",
+    "managedInstanceName": "myManagedInstanceName",
+    "databaseName": "myDatabase",
+    "schemaName": "dbo",
+    "api-version": "2018-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/myRG/providers/Microsoft.Sql/managedInstances/myManagedInstanceName/databases/myDatabase/schemas/dbo",
+        "name": "dbo",
+        "type": "Microsoft.Sql/managedInstances/databases/schemas",
+        "properties": {}
```

Comment:
Since the resource has nothing in the properties and the type doesn't explicitly say `"additionalProperties": true`,
 you need to remove this. This is what is causing the CI failure.